hail/python/hail/experimental/interact.py

```diff
@@ -190,121 +190,121 @@ def get_type_html(t):
+               f'<li>inherited class: {html_link(" Expression", "https://hail.is/docs/0.2/hail.expr.Expression.html")}</li>' \
                f'</ul>\n'
     elif isinstance(t, hl.tarray):
         if hl.expr.types.is_numeric(t.element_type):
             return f'<p>A variable-length array of homogenous numeric values.</p>\n' \
                    f'Documentation:\n<ul>' \
-                   f'<li>class: {html_link(" ArrayNumericExpression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.ArrayNumericExpression")}</li>' \
-                   f'<li>inherited class: {html_link(" ArrayExpression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.ArrayExpression")}</li>' \
-                   f'<li>inherited class: {html_link(" CollectionExpression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.CollectionExpression")}</li>' \
-                   f'<li>inherited class: {html_link(" Expression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.Expression")}</li>' \
+                   f'<li>class: {html_link(" ArrayNumericExpression", "https://hail.is/docs/0.2/hail.expr.ArrayNumericExpression.html")}</li>' \
+                   f'<li>inherited class: {html_link(" ArrayExpression", "https://hail.is/docs/0.2/hail.expr.ArrayExpression.html")}</li>' \
+                   f'<li>inherited class: {html_link(" CollectionExpression", "https://hail.is/docs/0.2/hail.expr.CollectionExpression.html")}</li>' \
+                   f'<li>inherited class: {html_link(" Expression", "https://hail.is/docs/0.2/hail.expr.Expression.html")}</li>' \
                    f'</ul>\n' \
                    f'<p>Access elements using square brackets: {html_code("x[i]")} or slice using Python syntax: {html_code("x[:end], x[start:], x[start:end]")}</p>'
         else:
             return f'<p>A variable-length array of homogenous values.</p>\n' \
                    f'Documentation:\n<ul>' \
-                   f'<li>class: {html_link(" ArrayExpression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.ArrayExpression")}</li>' \
-                   f'<li>inherited class: {html_link(" CollectionExpression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.CollectionExpression")}</li>' \
-                   f'<li>inherited class: {html_link(" Expression", "https://hail.is/docs/0.2/expressions.html#hail.expr.expressions.Expression")}</li>' \
+                   f'<li>class: {html_link(" ArrayExpression", "https://hail.is/docs/0.2/hail.expr.ArrayExpression.html")}</li>' \
+                   f'<li>inherited class: {html_link(" CollectionExpression", "https://hail.is/docs/0.2/hail.expr.CollectionExpression.html")}</li>' \
+                   f'<li>inherited class: {html_link(" Expression", "https://hail.is/docs/0.2/hail.expr.Expression.html")}</li>' \
```

Comment:
This loses the scrollto #expression. We could do `https://hail.is/docs/0.2/hail.expr.Expression.html#expression`. Similar for others.

I also noticed that while I fixed section scrollto, I did not for methods

Reply:
they were previously all in one page -- now that they have separate pages, we don't need a scrollto.